gr-digital/include/gnuradio/digital/constellation.h

```diff
@@ -206,7 +206,7 @@ class DIGITAL_API constellation : public boost::enable_shared_from_this<constell
     unsigned int d_arity;
     //! The factor by which the user given constellation points were
     //! scaled by to achieve an average amplitude of 1.
-    float d_scalefactor;
+    float d_scalefactor = 1.0;
```

Comment:
why this variable and not the others such as `d_dimensionality` or `d_re_min` and so forth? I see a pattern in the prior changes ... I don't see it here!

Reply:
I only took the one that was left out and not initialized, but yes better to take them all where they are common or have an obvious default. (e.g. scale 1.0)

Pushing new version of commit momentarily.

Reply:
Much better thx!